cgmml/models/HRNET/body_pose.py

```diff
@@ -108,9 +108,11 @@ def get_child_image(self, should_anonymize: bool, should_reorient: bool) -> np.a
         # Blur face
         if not should_anonymize:
             return image
-        highest_point = self.get_person_joints()[JOINT_INDEX_NOSE][0:3]
-        highest_point[1] += self.floor
-        image = blur_face(image, highest_point, self.dmap, CHILD_HEAD_HEIGHT_IN_METERS / 2.0)
+        eyes_center = self.get_person_joints()[JOINT_INDEX_LEFT_EYE][0:3]
+        eyes_center += self.get_person_joints()[JOINT_INDEX_RIGHT_EYE][0:3]
+        eyes_center /= 2.0
+        eyes_center[1] += self.floor
```

Comment:
[nit] Suggestion for slightly improved readability 

```suggestion
        left_eye = self.get_person_joints()[JOINT_INDEX_LEFT_EYE][0:3]
        right_eye = self.get_person_joints()[JOINT_INDEX_RIGHT_EYE][0:3]
        eyes_center = (left_eye + right_eye) / 2.0
        eyes_center[1] += self.floor
```